src/command.js

```diff
@@ -35,7 +35,7 @@ const validateArguments = (filter, json, options) => {
   const validatedPreSpawn = Joi.attempt(
     context,
     preSpawnSchema.tailor(validatedOptions.input),
-    { messages }
+    { messages, errors: { wrap: { label: '' } } }
```

Comment:
This value is `'"'` by default in v17, so set to blank string to keep the error message is still same.
https://joi.dev/api/?v=17.4.0#anyvalidatevalue-options